Dockerfile

```
@@ -26,7 +26,20 @@ ENV \
   RABBIT_HOST="rabbit:5672" \
   REDIS_HOST="redis:6379" \
   SAML_AUTH_ENABLE="False" \
-  SAML_AUTH_JSON_CONFIG=""
+  SAML_AUTH_JSON_CONFIG=""\
```

A default for `SLACK_INTEGRATION_ENABLE` would be good to have here.
```suggestion
  SAML_AUTH_JSON_CONFIG="" \
  SLACK_INTEGRATION_ENABLE="False" \
```

---

ok